### PR TITLE
Update requirements, move to pip-compile, remove unused nose params

### PR DESCRIPTION
**Pull request by bmedx**
Another step closer to consistency with other repos. Still needs PyPI integration with Travis. Otherwise this gets us pretty close.

### CODE REVIEW

File: workbench/test/selenium_test.py [hunk 1]
```diff
@@ -10,7 +9,6 @@
 from workbench.runtime_util import reset_global_state
 
 
-@attr('selenium')
```
Comment (jmbowman) [inline workbench/test/selenium_test.py]: Do you want to change this to a pytest mark instead?  (pytest can use the nose attrib pligin attrs also, but I'm all for gradually ripping out nose usage.)

Comment (bmedx) [reply workbench/test/selenium_test.py]: I wasn't sure we were actually using them for anything, but I guess it makes sense to keep them around for running those tests in isolation. I'll do that.

File: Makefile [hunk 2]
```diff
@@ -1,21 +1,26 @@
...
+	from urllib import pathname2url
+except:
+	from urllib.request import pathname2url
 
-.PHONY: install
-install: pip $(SQLITE_DB)
+webbrowser.open("file://" + pathname2url(os.path.abspath(sys.argv[1])))
+endef
+export BROWSER_PYSCRIPT
+BROWSER := python -c "$$BROWSER_PYSCRIPT"
```
Comment (doctoryes) [inline Makefile]: Fancy - TIL.

File: Makefile [hunk 3]
```diff
@@ -1,21 +1,26 @@
...
-	# we can install all requirements using setup.py
-	pip install -r requirements/base.txt
-	pip install -r requirements/django.txt
-	pip install -e .
-	pip install -r requirements/test.txt
+SQLITE_DB=var/workbench.db
+
+help: ## display this help message
+	@echo "Please use \`make <target>' where <target> is one of"
+	@perl -nle'print $& if m{^[a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
```
Comment (doctoryes) [inline Makefile]: Sneaking in the perl! 🐫

**Pull request merged**